app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -1,9 +1,9 @@
-<%= presenter.attribute_to_html(:nested_ordered_title_label, render_as: :scholars_archive_nested, search_field: 'nested_ordered_title_label', itemprop_option: 'title', label: 'Title') %>
+<%= presenter.attribute_to_html(:nested_ordered_title_label, render_as: :scholars_archive_nested, search_field: 'nested_ordered_title_label', itemprop_option: 'url', label: 'Title') %>
 <%= presenter.attribute_to_html(:alt_title, render_as: :external_link, label: t('simple_form.labels.defaults.alt_title')) %>
-<%= presenter.attribute_to_html(:nested_ordered_creator_label, render_as: :scholars_archive_nested, search_field: 'nested_ordered_creator_label', itemprop_option: 'creator', label: "Creator") %>
+<%= presenter.attribute_to_html(:nested_ordered_creator_label, render_as: :scholars_archive_nested, search_field: 'nested_creator', itemprop_option: 'url', label: "Creator") %>
```

Comment:
Does this change maintain the order set in `nested_ordered_creator`? I think it would be a good idea to define a spec for this case in https://github.com/osulp/Scholars-Archive/blob/master/spec/renderers/hyrax/renderers/scholars_archive_nested_attribute_renderer_spec.rb to check the order as well in the renderer.